Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1978,6 +2035,69 @@ def get_white_list_command(client: Client, args: dict) -> CommandResults:
         raw_response=exclusion_items)
 
 
+def get_item_ids_from_whitelist(client: Client, item: str, exclusion_type: str, os_type: str = None) -> List[Optional[str]]:
+    """
+    Return the IDs of the hash from the white. Helper function for remove_item_from_whitelist
+
+    A hash can occur more than once if it is blocked on more than one platform (Windwos, MacOS, Linux)
+    """
+    item_ids = []
+    limit = 4
```

Comment:
This function is basically a duplicate of the get_hash_ids_from_blocklist functions, the limit here is 4 because it is only supposed to be able to return 3 in any case because the hash will only return multiple times if it is whitelisted on multiple platforms and there are only 3 different OS types where it can be whitelisted so if it returns 4 items we know it is an error.